Procurement/Controls/ItemDisplay.xaml.cs

```diff
@@ -68,9 +86,16 @@ private void ItemDisplay_Loaded(object sender, RoutedEventArgs e)
             {
                 var i = vm.getImage();
                 itemImage = i;
-
+                
                 if (i != null)
                 {
+                    //See: https://github.com/Stickymaddness/Procurement/issues/966
```

Comment:
Include a summary here of the situation, as well as the reference. (The link might not always be valid.)